client/js/app/modules/api_query_url.js

```diff
@@ -0,0 +1,71 @@
+var _ = require('lodash');
+var Qs = require('qs');
+var ExplorerUtils = require('../utils/ExplorerUtils');
+var TimeframeUtils = require('../utils/TimeframeUtils');
+
+function ApiQueryUrl(client, explorer) {
```

Comment:
Would this still belong in `modules`, then? It almost feels like it would belong in `components/explorer/query_builder`, except everything else in that file is a react component.